GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMesh.java

```diff
@@ -362,14 +362,6 @@ public GVRMesh getBoundingBox() {
         return mBones;
     }
     /**
```

Comment:
So what change makes the need to distinguish static vs dynamic mesh not needed anymore? How is the dynamic case handled now?